csv-data-management/playbooks/playbooks.json

```diff
@@ -501,11 +501,110 @@
           ],
           "priority": "/api/3/picklists/2b563c61-ae2c-41c0-a85a-c9709585e3f2",
           "uuid": "baccb6ab-fbf1-4847-9942-81a10da6d9e1",
-          "id": 1371,
-          "createUser": "/api/3/people/ffd74c31-e057-4bd5-a395-30303223ed6e",
-          "createDate": 1650951180.042837,
-          "modifyUser": "/api/3/people/ffd74c31-e057-4bd5-a395-30303223ed6e",
-          "modifyDate": 1650964850.479053,
+          "id": 656,
+          "createUser": "/api/3/appliances/57545210-2adc-472b-a24f-2df6ee8dfe63",
+          "createDate": 1651750119.115468,
+          "modifyUser": "/api/3/people/3451141c-bac6-467c-8d72-85e0fab569ce",
+          "modifyDate": 1652158164.99216,
+          "owners": [],
+          "isPrivate": false,
+          "deletedAt": null,
+          "importedBy": [],
+          "recordTags": []
+        },
+        {
+          "@type": "Workflow",
+          "triggerLimit": null,
+          "name": "Read two CSV and join in single data set",
```

Comment:
Read two CSVs and join into single data set

Reply:
updated